include/path.hpp

```diff
@@ -0,0 +1,141 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Core>
+#include <glm/glm.hpp>
+
+
+// -------------------------------------------------------------------------------------------------------------------//
+
+// Misc definitions
+
+
+// Flags for boundary conditions
+// Since Eigen arrays don't work well with enum types, we just take integer flags
+typedef int bc_t;
+static const bc_t BC_T_START = 0, BC_ZERO = 0, BC_NATURAL = 1, BC_CLAMP = 2, BC_T_END = 3;
```

Comment:
Hm, I actually tried that at some point, but Eigen would emit errors. Probably something else was wrong at that time, because now it works..